### PR TITLE
feat [3/4]: Update invoker to use api-common-protos

### PR DESCRIPTION
**Pull request by xiaozhenliu-gg5**
If the `api-common-protos` is set to be true, then the protoc command should include `api-common-protos` and protobuf protos in `test/tools/protos` as proto path folder.

### CODE REVIEW

**Review by software-dov — APPROVED** [review]

**Comment by software-dov** [comment]
I think this repo doesn't have protection set up; the "Squash and merge" button was green and clickable before I reviewed the changes.

**Comment by xiaozhenliu-gg5** [comment]
Oh, it's acting different from my end, without an approval, the `Squash and merge` button is grey and not clickable for me. I set the `@googleapis/actools` to be the CODEOWNERS, that might be the reason why on your side it can be merged? @software-dov Thanks for reporting this, I will keep an eye this if it's not actingn properly for the branch protection.

**Pull request merged**
